tests/test_implements_satisfy_protocols.py

```diff
@@ -23,22 +23,22 @@
     NodeRelation,
     PydanticSemanticModel,
 )
-from dbt_semantic_interfaces.protocols.dimension import Dimension as DimensionProtocol
-from dbt_semantic_interfaces.protocols.entity import Entity as EntityProtocol
-from dbt_semantic_interfaces.protocols.measure import Measure as MeasureProtocol
-from dbt_semantic_interfaces.protocols.metadata import Metadata as MetadataProtocol
-from dbt_semantic_interfaces.protocols.metric import Metric as MetricProtocol
-from dbt_semantic_interfaces.protocols.semantic_manifest import (
+from dbt_semantic_interfaces.protocols import Dimension as DimensionProtocol
```

Comment:
I'm unsure as to why, but for some reason `isort` keeps on correcting this from
```python
from dbt_semantic_interfaces.protocols import (
  Dimension as DimensionProtocol,
  Entity as EntityProtocol,
  Measure as MeasureProtocol,
  Metadata as MetadataProtocol,
  Metric as MetricProtocol,
  SemanticManifest as SemanticManifestProtocol,
  SemanticModel as SemanticModelProtocol,
)
```
to instead the individual imports 🤔

Reply:
Maybe this configuration https://beta.ruff.rs/docs/settings/#isort-combine-as-imports